python_modules/dagster/dagster/_core/definitions/declarative_scheduling/scheduling_condition.py

```diff
@@ -211,6 +211,9 @@ def eager() -> "SchedulingCondition":
         - None of its parent partitions are missing
         - None of its parent partitions are currently part of an in-progress run
         - It is not currently part of an in-progress run
+
+        It will also refuse to materialize an asset partition if the latest materialization attempt
+        failed and has not already been scheduled within the last hour, to prevent repeated failures.
```

Comment:
I think this is arbitrary enough that we should make it optionally parametrizable in the `eager` constructor.